bot/cogs/commands.py

```diff
@@ -52,8 +53,7 @@ async def serverinfo(self, ctx: Context) -> None:
     async def userinfo(self, ctx: Context, user: t.Optional[t.Union[Member, User]] = None) -> None:
         """
         Get information about you, or a specified member.
-
-        `user`: The user to get information from. Can be a Mention, Name or ID.
+        'user' can be a user Mention, Name, or ID.
```

Comment:
Actually we insist on using \`user\` instead of 'user' as \` is a symbol which marks inline code in discord markdown and this help docstring will be displayed in a help command for this function so we should keep the user wrapped in backticks instead of single quotes